examples/abitest/client/abitest.cc

```diff
@@ -158,9 +160,30 @@ void run_grpc_test_server(int grpc_test_port, grpc::Server** grpc_test_server) {
 
 }  // namespace
 
+std::shared_ptr<grpc::ServerCredentials> create_grpc_credentials(std::string ca_cert) {
+  std::string private_key_path = absl::GetFlag(FLAGS_private_key);
+  std::string cert_chain_path = absl::GetFlag(FLAGS_cert_chain);
+  if (private_key_path.empty()) {
+    OAK_LOG(FATAL) << "No private key file specified.";
+  }
+  if (cert_chain_path.empty()) {
+    OAK_LOG(FATAL) << "No certificate chain file specified.";
+  }
+  std::string private_key = oak::utils::read_file(private_key_path);
+  std::string cert_chain = oak::utils::read_file(cert_chain_path);
+
+  grpc::SslServerCredentialsOptions::PemKeyCertPair key_cert_pair = {private_key, cert_chain};
+  grpc::SslServerCredentialsOptions options;
+  options.pem_root_certs = ca_cert;
+  options.pem_key_cert_pairs.push_back(key_cert_pair);
+  return grpc::SslServerCredentials(options);
```

Comment:
Presumably this is more because the Rust gRPC *client* pseudo-Node now always uses TLS, so the test server that it connects to (which runs in parallel inside the `abitest` client program) now needs to act as TLS server side?